vite.config.js

```diff
@@ -0,0 +1,38 @@
+import laravel from 'laravel-vite-plugin'
+import vue from '@vitejs/plugin-vue'
+import { defineConfig } from 'vite'
+
+export default defineConfig({
+    plugins: [
+        vue({
+            template: {
+                transformAssetUrls: {
+                    // The Vue plugin will re-write asset URLs, when referenced
+                    // in Single File Components, to point to the Laravel web
+                    // server. Setting this to `null` allows the Laravel plugin
+                    // to instead re-write asset URLs to point to the Vite
+                    // server instead.
+                    base: null,
+
+                    // The Vue plugin will parse absolute URLs and treat them
+                    // as absolute paths to files on disk. Setting this to
+                    // `false` will leave absolute URLs un-touched so they can
+                    // reference assets in the public directory as expected.
+                    includeAbsolute: false,
+                },
+            },
+        }),
+        laravel(['resources/css/app.css', 'resources/js/app.js']),
+        {
+            name: 'blade',
+            handleHotUpdate({ file, server }) {
+                if (file.endsWith('.blade.php')) {
+                    server.ws.send({
+                        type: 'full-reload',
+                        path: '*',
+                    })
+                }
+            },
```

Comment:
```suggestion
            },
            valetTls: 'atlas.sng.local',
```

since I use valet for local development, I am facing here issues with specified here: https://laravel.com/docs/9.x/vite#working-with-a-secure-development-server

Reply:
Could you look into making the value configurable in a new issue?